projects/novo-elements/src/elements/picker/extras/grouped-multi-picker-results/GroupedMultiPickerResults.ts

```diff
@@ -275,7 +292,14 @@ export class GroupedMultiPickerResults extends BasePickerResults implements OnIn
     if (this.searchTerm && this.searchTerm.length !== 0 && this.selectedCategory) {
       matches = matches.filter((match) => {
         const searchTerm = this.searchTerm.toLowerCase();
-        return match.label.toLowerCase().indexOf(searchTerm) > -1 || match.value.toLowerCase().indexOf(searchTerm) > -1;
+        return (
+          match.label.toLowerCase().indexOf(searchTerm) > -1 ||
+          match.value
+            .toLowerCase()
+            .split('.') // Split into to array since the first part of the string (of the format '[entityType].[value]') will break the filter functionality
+            .pop() // Only the last element of the array should be checked by the searchTerm filter, e.g. if the value is 'placement.FirstName' then 'placement' is not relevant to the users search
```

Comment:
why are we always splitting the value by `.`?  if the match condition is different for the type of data you have with the picker, the match function should be overridden on the client side (in novo)